src/InputGuess.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Input from "@mui/material/Input";
+import { Button } from "@mui/material";
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userWord: "",
+    };
+  }
+
+  handleChange = (e) => {
+    let { value } = e.target;
+    if ((value.match(/^[a-z]+$/) && value.length === 1) || value === "") {
+      this.setState({ userWord: value });
+    }
+  };
+
+  handleSubmit = (e) => {
+    let { userWord } = this.state;
+    e.preventDefault();
+    if (userWord === "") {
+      return;
+    } else {
+      this.props.updateGuess(userWord);
+      this.setState({
+        userWord: "",
+      });
+    }
```

Comment:
Since you return early, we have no need for the else statement. Skip the curly braces and you save some lines and improve readability.

```suggestion
    if (!userWord) return
    this.props.updateGuess(userWord);
    this.setState({
        userWord: "",
      });
```